lib/src/signed_video_internal.h

```diff
@@ -153,7 +153,7 @@ struct _signed_video_t {
   unsigned recurrence_offset;
 
   // Frame counter and flag to handle recurrence in frames functionality
-  bool nr_of_frames_for_recurrence_passed;
+  bool add_recurrent_data;
```

Comment:
Changed my mind. `recurrent_data_ready` or simply `has_recurrent_data`

When you ready `self->has_recurrent_data` it is self desriptive.